app/src/marketplace/modules/createContractProduct/services.js

```diff
@@ -28,7 +28,7 @@ const createOrUpdateContractProduct = (method: (...any) => Sendable, product: Sm
     validateProductPriceCurrency(priceCurrency)
     const transformedPricePerSecond = mapPriceToContract(pricePerSecond)
     const methodToSend =
-        method(getValidId(id), name, beneficiaryAddress, transformedPricePerSecond, currencyIndex, minimumSubscriptionInSeconds)
+        method(getValidId(id), name, beneficiaryAddress, transformedPricePerSecond, currencyIndex, minimumSubscriptionInSeconds, false)
```

Comment:
This extra boolean parameter was added in new Marketplace contract. If true, it will publish the product if unpublished. It will save one transaction but the publish process needs to be updated to make use of it.

Reply:
Works well 👍